build.gradle

```diff
@@ -74,7 +75,13 @@ task( compileTestTaxonomiesInflection, dependsOn: compileTestModelJava, type: Ja
 	main = "ch.liquidmind.inflection.compiler.InflectionCompiler"
 	classpath = sourceSets.main.runtimeClasspath + sourceSets.testModel.runtimeClasspath
 	jvmArgs '-ea'
-	args "${buildDir}/resources/testModel", "BOOTSTRAP", "src/test/resources/ch/liquidmind/inflection/test/model/TestModel.inflect"
+	args "${buildDir}/resources/testModel", "BOOTSTRAP", 
```

Comment:
We could do it in @BeforeClass or within the test method itself, depending on the situation. Performance could be an issue. I suggest to think about this and implement it in an subsequent branch.